cmd/cartesi-node/validator.go

```diff
@@ -3,12 +3,21 @@
 
 package main
 
-import "github.com/spf13/cobra"
+import (
+	"github.com/cartesi/rollups-node/internal/pkg/services"
+	"github.com/spf13/cobra"
+)
 
 var validator = &cobra.Command{
 	Use:   "validator",
 	Short: "Starts the node in validator mode",
-	Run: func(cmd *cobra.Command, args []string) {
-		println("TODO")
-	},
+	Run:   runValidatorNode,
+}
+
+func runValidatorNode(cmd *cobra.Command, args []string) {
+	validatorServices := []services.Service{
+		&services.GraphQLService{},
```

Comment:
```suggestion
		services.GraphQLService{},
```
We don't need to use the GraphQLService as a pointer.